vector/src/main/AndroidManifest.xml

```diff
@@ -46,6 +46,7 @@
     <uses-permission android:name="android.permission.ACCESS_COARSE_LOCATION" />
     <uses-permission android:name="android.permission.ACCESS_FINE_LOCATION" />
     <uses-permission android:name="android.permission.ACCESS_BACKGROUND_LOCATION" />
+    <uses-permission android:name="android.permission.FOREGROUND_SERVICE" />
```

Comment:
we already have this permission :smile:  https://github.com/vector-im/element-android/blob/develop/vector/src/main/AndroidManifest.xml#L13

Reply:
Oh no! Good catch, done.